source/initial_temperature/patch_on_S40RTS.cc

```diff
@@ -186,7 +186,7 @@ namespace aspect
                                               "contain the number of grid points in each dimension as "
                                               "for example '# POINTS: 3 3 3'. "
                                               "The order of the data columns has to be "
-                                              " `x', `y', `z', 'Vs Perturbation [%]' in a 3d model, which means that "
+                                              " `x', `y', `z', 'Vs Perturbation' in a 3d model, which means that "
```

Comment:
do we say anywhere that this column is interpreted as a percentage perturbation?